sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,282 @@
+# TBD'22 release notes
+
+Release notes for commit range [`4043dda3..0f579bae`](https://github.com/intel/llvm/compare/4043dda3...0f579bae)
+
+**TODO**
+
+- commit 2ffd50a8cf8aefec0582da0ab5908c1665dcdbb6
+- commit 7c58b9a63cc7797dea5bc4037e2ef880a16785e2
+
+## New features
+
+### SYCL Compiler
+
+- Added ability to enforce stateless memory accesses for ESIMD. [18111623]
+- Added support for `-fsycl-force-target` compiler option. [1d95f2ec]
+- Added support for `[[intel::max_reinvocation_delay`]] loop attribute. [90fa5bb0]
+- Added support for linking object files larger than 2GB by adding
+  `-fsycl-huge-device-code` compiler option. [f963062c]
+- Added support for compiling `.cu` files with SYCL compiler. [e76ad72f]
+- Added support for `assert` on HIP backend. [ade1870a]
+- Enabled CXX standard library functions for CUDA backend. [1fe92c54]
+- Implemented group collective built-in functions for more integral types. [d4933b6f]
+
+### SYCL Library
+
+- Implemented SYCL 2020 callable device selectors. [64f0db7a]
+- Implemented SYCL 2020 standalone device selectors. [bfc7e984]
+- Added SYCL 2020 property iterfaces for `local_accessor`, `usm_allocator`,
```

Comment:
```suggestion
- Added SYCL 2020 property interfaces for `local_accessor`, `usm_allocator`,
```

Reply:
Applied in ed5e71d464608b7be8fd28ee14249979e67e4fb8